pipeline/run.sh

```diff
@@ -168,7 +168,7 @@ done
 job_ids=\${job_ids::-1}
 echo \${job_ids}
 
-sbatch --job-name=1-queueStart_${name} --time=${queue_1_time} --mem=${queue_1_mem} --dependency=afterany:\${job_ids} --output=${outdir}/logs/queue/1-queueStart.o --error=${outdir}/logs/queue/1-queueStart.e ${global_sbatch_parameters} ${outdir}/jobs/queue/1-queueStart.sh
+sbatch --job-name=1-queueStart_${name} --time=${queue_1_time} --mem=${queue_1_mem} --dependency=afterany:\${job_ids}:+10 --output=${outdir}/logs/queue/1-queueStart.o --error=${outdir}/logs/queue/1-queueStart.e ${global_sbatch_parameters} ${outdir}/jobs/queue/1-queueStart.sh
```

Comment:
Did this change have any effect on the runs? Please explain why this was needed.

Reply:
I left it in. Might improve the runs (by decreasing the errors), but not a complete fix as shown in the testrun. This hotfix is not a fix for the problem yet, more 
1. A way to make the warnings visible for the user, while a fix is being made
2. When a fix is made, to see if it always keeps working (hard to validate, because it does not always happen)